sitemap/sitemap.ts

```diff
@@ -0,0 +1,84 @@
+import { merge } from "lume/core/utils.ts";
+import { Page } from "lume/core/filesystem.ts";
+import { buildFilter, buildSort } from "lume/plugins/search.ts";
+
+import type { Site } from "lume/core.ts";
+
+export interface Options {
+  /** The filters to search pages included in the sitemap */
+  filters: string[];
+
+  /** The values to sort the sitemap */
+  sort: string[];
+
+  /** The default filters including the index page (only if custom `filters` are defined) */
+  defaultFilters: string[];
```

Comment:
I don't understand why we need this. `query` (instead of filters) and `sort` is enough.

Reply:
If a user defines a custom query, the index page `["url=/"] is not included, if he does not specify. I thought this could be a way to let handle this issue.

Never mind I removed it, added a info about to the README.md and resolved the issue.